curvefs/src/metaserver/inode_manager.cpp

```diff
@@ -305,7 +305,8 @@ MetaStatusCode InodeManager::UpdateInode(const UpdateInodeRequest &request) {
     }
 
     if (needAddTrash) {
-        trash_->Add(old.fsid(), old.inodeid(), old.dtime());
+        trash_->Add(old->fsid(), old->inodeid(), old->dtime());
+        *isDeleted = true;
```

Comment:
The param `isDeleted` should assign to `false` by default.

Reply:
> The param `isDeleted` should assign to `false` by default.

fix